fanova/fanova.py

```diff
@@ -93,8 +105,13 @@ def __init__(self, X, Y, config_space=None,
             if isinstance(hp, CategoricalHyperparameter):
                 types[i] = len(hp.choices)
                 pcs[i] = (len(hp.choices), np.nan)
-            else:
+            elif isinstance(self.cs_params[i], NumericalHyperparameter):
                 pcs[i] = (hp.lower, hp.upper)
+            elif isinstance(self.cs_params[i], (UnParametrizedHyperparameter, Constant)):
+                pcs[i] = (hp.value, hp.value)
```

Comment:
I'm not sure if that is correct for constants.
I'd expect something more like
```suggestion
                pcs[i] = (1, np.nan)
                types[i] = 1
```
as you have only one choice and no upper limit.

But I don't see why this is necessary with the raised TypeError in Line86/Line93

Reply:
you are right. I will raise another `TypeError` for now.